WalletWasabi.Gui/Controls/WalletExplorer/TransactionBroadcasterViewModel.cs

```diff
@@ -165,18 +166,17 @@ private async Task OnDoTransactionBroadcastAsync()
 				ButtonText = "Broadcasting Transaction...";
 
 				SmartTransaction transaction;
-				try
-				{
-					var signedPsbt = PSBT.Parse(TransactionString, Global.Network ?? Network.Main);
 
+				if (PSBT.TryParse(TransactionString, Global.Network ?? Network.Main, out var signedPsbt))
```

Comment:
Can really `Global.Network` be null here? I am pretty sure it cannot.

Reply:
It was there before but i do think that's a good failsafe. even if it's redundant.

Reply:
It cannot be null.

Reply:
I'm ok with both approach.